src/docs/resources/faq.md

```diff
@@ -592,7 +596,406 @@ For more information, see the Medium article
+[example project]: {{site.github}}/flutter/flutter/tree/master/examples/platform_channel
+[Executing Dart in the Background with Flutter Plugins and Geofencing]: {{site.flutter-medium}}/executing-dart-in-the-background-with-flutter-plugins-and-geofencing-2b3e40a1a124
+[`FlatButton`]: {{site.api}}/flutter/material/FlatButton-class.html
+[Flutter 1.0: Google's Portable UI Toolkit]: https://developers.googleblog.com/2018/12/flutter-10-googles-portable-ui-toolkit.html
+[flutter_view]: {{site.github}}/flutter/flutter/tree/master/examples/flutter_view
+[`Future`]: {{site.api}}/flutter/dart-async/Future-class.html
+[Get $75 app advertising credit when you spend $25.]: https://ads.google.com/lp/appcampaigns/#?modal_active=none&subid=ww-ww-et-aw-a-flutter1!o3
+[gesture system]: /docs/development/ui/advanced/gestures
+[`GestureDetector`]: {{site.api}}/flutter/widgets/GestureDetector-class.html
+[GitHub]: {{site.github}}/flutter/flutter
+[`GlobalKey`]: {{site.api}}/flutter/widgets/GlobalKey-class.html
+[Hamilton for Android]: https://play.google.com/store/apps/details?id=com.hamilton.app
+[Hamilton for iOS]: https://itunes.apple.com/us/app/hamilton-the-official-app/id1255231054?mt=8
+[hot reload]: #hot-reload
+[Hot reload]: /docs/development/tools/hot-reload
+[`icon`]: {{site.api}}/flutter/widgets/Icon/icon.html
+[`Icon`]: {{site.api}}/flutter/widgets/Icon-class.html
+[`IconTheme`]: {{site.api}}/flutter/widgets/IconTheme-class.html
+[`InkWell`]: {{site.api}}/flutter/material/InkWell-class.html
+[iOS]: #run-ios
+[iOS App Store Specific Considerations]: https://developer.apple.com/library/archive/qa/qa1795/_index.html#//apple_ref/doc/uid/DTS40014195-CH1-APP_STORE_CONSIDERATIONS
+[iOS instructions]: /docs/deployment/ios
+[IntelliJ IDEA]: https://www.jetbrains.com/idea/
+[internationalization tutorial]: /docs/development/accessibility-and-localization/internationalization
+[is easy]: /docs/development/packages-and-plugins/using-packages
+[issue #14821]: {{site.github}}/flutter/flutter/issues/14821
+[issue #9253]: {{site.github}}/flutter/flutter/issues/9253
+[issue tracker]: {{site.github}}/flutter/flutter/issues
+[issues related to running Flutter on Chromebooks]: {{site.github}}/flutter/flutter/labels/platform-arc
+[`Iterable`]: {{site.api}}/flutter/dart-core/Iterable-class.html
+[license file]: https://raw.githubusercontent.com/flutter/engine/master/sky/packages/sky_engine/LICENSE
+[`LicenseRegistry`]: {{site.api}}/flutter/foundation/LicenseRegistry-class.html
+[`List]`: {{site.api}}/flutter/dart-core/List-class.html
+[`Listenable`]: {{site.api}}/flutter/foundation/Listenable-class.html
+[`Map`]: {{site.api}}/flutter/dart-core/Map-class.html
+[`Material`]: {{site.api}}/flutter/material/Material-class.html
+[`MaterialButton`]: {{site.api}}/flutter/material/MaterialButton-class.html
+[MDC-103 Flutter: Material Theming]: {{site.codelabs}}/codelabs/mdc-103-flutter
```

Comment:
This link isn't working, for some reason. (I see "][]" in the place where it's used.)